data/ansible/domain.yml

```
@@ -12,3 +12,15 @@ slots: {}
 responses:
   utter_ansible_inform_reach_out_to_support:
     - text: "I'm sorry you are having trouble accessing a namespace. Please reach out to [partner-help@redhat.com](mailto:partner-help@redhat.com) for any questions about the namespace access."
+  utter_ansible_inform_assist_with_free_trial:
+    - text: |
+        You can get access to a free trial of the Ansible Automation Platform here [https://console.redhat.com/ansible/ansible-dashboard](https://console.redhat.com/ansible/ansible-dashboard).
+
+        Wondering, why you should convert to premium? Click here to learn more: [https://www.redhat.com/en/about/value-of-Red-Hat](https://www.redhat.com/en/about/value-of-Red-Hat)
+  utter_ansible_inform_automation_calculator:
+    - text: |
+        The automation calculator provides graphs, metrics, and calculations that help you determine the total savings on your investment in automated process.
+
+        Here is some documentation to learn more about the Automation Calculator: [https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.1/html-single/using_the_automation_calculator/index](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.1/html-single/using_the_automation_calculator/index)
```

For these links, ik they are like that in the miro. Can you change them to not show the url? Users don't find that helpful usually

so this one could be

```suggestion
        Here is some documentation to learn more about the Automation Calculator: [Using the automation calculator](https://docs.redhat.com/en/documentation/red_hat_ansible_automation_platform/2.1/html-single/using_the_automation_calculator/index)
```

for the other ones you can put the hyperlinks around `here`